app/views/admin/admin_users/show.html.erb

```diff
@@ -35,6 +35,26 @@
       <dd><%= @user.invitation ? link_to(@user.invitation.id, invitation_path(@user.invitation.id)) : t(".info.no_invitation") %></dd>
       <dt><%= t(".info.role", count: @user.roles.size) %></dt>
       <dd><%= @user.roles.any? ? @user.roles.map { |role| t("activerecord.attributes.role.#{role.name}") }.to_sentence : t(".info.no_role") %></dd>
+      <% if policy(User).can_view_past? %>
+        <% past_logins = @user.historic_values("login") %>
+        <% if past_logins.present? %>
+          <dt><%= t(".info.past_logins", count: past_logins.size) %></dt>
+          <dd>
+            <ul>
+              <li><%= past_logins.join(", ") %></li>
```

Comment:
Can we use `to_sentence` for this and emails instead? iirc, it's easier to localize than hard coding the comma with `join`. I'm also not sure the `<ul>` and `<li>` are necessary, since it's not like each login/email is its own item.